login.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet"  type="text/css" href="application.css">
+  </head>
+  <body>
+    <div class="main-container">
+      <form class="login-container" action="books-list.html">
+        <div class="login-elements">
+          <img src="images/w-logo.png" alt="wolox_logo" class="w-logo">
+          <label class="data-label" for="email">Email</label>
+          <input class="data-input" type="email" id="email">
+          <label class="data-label" for="password">Password</label>
+          <input class="data-input" type="password" id="password">
+          <button class="generic-btn login-btn" type="submit">Login</button>
+        </div>
+        <button class="generic-btn sign-up-btn" type="button">Sign Up</button>
```

Comment:
Shouldn't this be a link? 🤔

Reply:
To be honest, neither the card or the inVision design clarify that, there's no part where I could assume this is not a button, so I took it as a button, like the login button. I could change it if you want, but do you think is necessary, given that there's nothing that clarifies it?

Reply:
No, it's OK. If there's no sign up view, don't mind